tests/devices/qubit/test_apply_operation.py

```diff
@@ -1329,8 +1329,8 @@ def test_conditional(self, wires, unitary, batched, ml_framework):
             qml.math.squeeze(initial_state), qml.math.reshape(new_state, (n_states, 4))
         )
 
-    @pytest.mark.parametrize("rng_seed, m_res", ((12, (0, 0)), (42, (1, 1))))
-    def test_mid_measure(self, rng_seed, m_res):
+    @pytest.mark.parametrize("m_res", [(0, 0), (1, 1)])
```

Comment:
This test was initially seeded, and the seeds were used to force the expected result out of `numpy.random.binomial` for each test case. I feel that this is a perfect example of when patching is more suitable than seeding.

Reply:
@mudit2812 This is the test I mentioned.